ayu.md

```diff
@@ -0,0 +1,95 @@
+# Ayu Themes
+
+![AyuMirage](images/AyuMirage.png)
+![AyuDark](images/AyuDark.png)
+![AyuLight](images/AyuLight.png)
+
+## Credit
+
+[Theme: Ayu Theme](https://github.com/ayu-theme)
+
+[Scheme creator: joshtynjala](https://gist.github.com/joshtynjala/10dbdd4e449027fe9723b1d9f553c0bd)
+
+## Settings
+
+```json
+{
+    "colorScheme": "AyuMirage"
```

Comment:
The `colorScheme` doesn't match the ones defined, should be `Ayu Mirage` instead.

Reply:
done.

Reply:
Thank you!